Gruntfile.coffee

```diff
@@ -1061,6 +1063,20 @@ module.exports = (grunt) ->
 						return dest + "/" + src.replace ".debug", ""
 					expand: true
 					flatten: true
+				,
+					cwd: "lib/MathJax"
+					src: [
+						"MathJax.js"
+						"config/**"
+						"docs/**"
```

Comment:
Should the docs be included? When `bootlint` gets run from the `Gruntfile.coffee`, it fails on the HTML files copied over into this directory. I suspect these files aren't even required and could be removed to easily resolve this issue.

Reply:
@Empereol Please submit a [new issue](https://github.com/wet-boew/wet-boew/issues/new) with your concern. That way it is easier to track and ensure that you will get a follow up.

Reply:
Can do. Issue #8250